core-web/libs/contenttype-fields/src/lib/fields/binary-field/binary-field.component.ts

```diff
@@ -0,0 +1,31 @@
+import { MonacoEditorModule } from '@materia-ui/ngx-monaco-editor';
+
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+import { DialogModule } from 'primeng/dialog';
+
+import { DotMessagePipe } from '@dotcms/ui';
+
+@Component({
+    selector: 'dotcms-binary-field',
+    standalone: true,
+    imports: [ButtonModule, DialogModule, MonacoEditorModule, DotMessagePipe],
+    templateUrl: './binary-field.component.html',
+    styleUrls: ['./binary-field.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class BinaryFieldComponent implements OnInit {
+    public labelCode!: string;
+    visible = false;
+
+    constructor(private dotMesagePipe: DotMessagePipe) {}
+
+    ngOnInit() {
+        this.labelCode = this.dotMesagePipe.transform('contenttypes.content.edit.write.code');
```

Comment:
why don't we use the pipe directly or `DotMessageService`?

Reply:
The label is being used in the button and in the header. That's why I created a variable.